packages/flutter/lib/src/material/selectable_text.dart

```diff
@@ -256,6 +263,11 @@ class SelectableText extends StatefulWidget {
     assert(autofocus != null),
     assert(dragStartBehavior != null),
     assert(maxLines == null || maxLines > 0),
+    assert(minLines == null || minLines > 0),
```

Comment:
Duplicate assert

Reply:
But this is for minlines not maxlines

Reply:
Oops, sorry about that! Please disregard my comment.

Reply:
no problem